lumigator/python/mzai/backend/backend/config_templates.py

```diff
@@ -134,7 +134,8 @@
     "job": {{
         "max_samples": {max_samples},
         "storage_path": "{storage_path}",
-        "output_field": "{output_field}"
+        "output_field": "{output_field}",
+        "enable_tqdm": true
```

Comment:
Need this because the schema doesn't have a default of True. I could change the schema so that then this tqdm key doesn't need to be added